index.js

```diff
@@ -29,15 +29,28 @@ function configureAxe (defaultOptions = {}) {
       throw new Error(`html parameter ("${html}") has no elements`)
     }
 
+    // Before we use Jests's jsdom document we store and remove all child nodes from the body.
+    const bodyChilds = document.createElement('div');
+    document.body.childNodes.forEach(child => {

```

Comment:
This is looking good.

Is there anything funky that could happen e.g. unexpected execution of JavaScript?